launcher/ui/pages/instance/ModFolderPage.cpp

```diff
@@ -86,12 +88,30 @@ ModFolderPage::ModFolderPage(BaseInstance* inst, std::shared_ptr<ModFolderModel>
         ui->actionsToolbar->insertActionAfter(ui->actionAddItem, ui->actionUpdateItem);
         connect(ui->actionUpdateItem, &QAction::triggered, this, &ModFolderPage::updateMods);
 
+        ui->actionVisitItemPage->setToolTip(tr("Go to mod's home page"));
+        ui->actionsToolbar->insertActionAfter(ui->actionViewFolder, ui->actionVisitItemPage);
+        connect(ui->actionVisitItemPage, &QAction::triggered, this, &ModFolderPage::visitModPages);
+
         auto check_allow_update = [this] {
             return (!m_instance || !m_instance->isRunning()) && (ui->treeView->selectionModel()->hasSelection() || !m_model->empty());
         };
 
-        connect(ui->treeView->selectionModel(), &QItemSelectionModel::selectionChanged, this,
-                [this, check_allow_update] { ui->actionUpdateItem->setEnabled(check_allow_update()); });
+        connect(ui->treeView->selectionModel(), &QItemSelectionModel::selectionChanged, this, [this, check_allow_update] {
+            ui->actionUpdateItem->setEnabled(check_allow_update());
+
+            auto selection = m_filterModel->mapSelectionToSource(ui->treeView->selectionModel()->selection()).indexes();
+            auto mods_list = m_model->selectedMods(selection);
+            auto selected = std::count_if(mods_list.cbegin(), mods_list.cend(),
+                                          [](Mod* v) { return v->metadata() != nullptr || v->homeurl().size() != 0; });
+            if (selected <= 1) {
+                ui->actionVisitItemPage->setText(tr("Visit mod's page"));
+                ui->actionVisitItemPage->setToolTip(tr("Go to mod's home page"));
+            } else {
+                ui->actionVisitItemPage->setText(tr("Visit the pages of the selected mods"));
+                ui->actionVisitItemPage->setToolTip(tr("Go to the pages of the selected mods"));
+            }
```

Comment:
Inconsistent with other actions i guess

Reply:
Can you please suggest a more general language for both cases: when one mod is selected or when are multiple?

Reply:
not sure it's clear what's inconsistent here... but i'll try to provide some suggestions to at least get this thing rolling!

Selected one:
* setText - View mod website
* setTooltip - Opens mod website in browser

Multiple:
* setText - View mod websites
* setTooltip - Opens selected mod websites in browser

Reply:
Oops. My inbox kind of gets cluttered with notifications :P.

What I meant was other actions don't have dynamic text.

Reply:
Hmm, no harm in having it work like this I guess. The other actions should be changed in future though!

Reply:
ahh makes sense!

Reply:
Hmm, on second thoughts, won't this look pretty long in the ui?

Reply:
how about:
`Visit mod's page`
`Visit mods pages`
very short and simple
(`mod's` or `mods`, idk.. english isn't my first language x3)